mods/ra/rules/aircraft.yaml

```diff
@@ -2,13 +2,14 @@ BADR:
 	Inherits: ^NeutralPlane
 	ParaDrop:
 		DropRange: 4c0
+		DropInterval: 5
```

Comment:
This is already set as the default value.

Reply:
My bad, I needed to change this value when I was testing out faster badger speeds, but I never looked up the original one. It also seems that paradrop behaviour remains the same despite the speed increase